config

```diff
@@ -51,10 +51,16 @@ if test -n "$ngx_module_link"; then
     ngx_module_deps="$ECHO_DEPS"
     ngx_module_srcs="$ECHO_SRCS"
     ngx_module_libs=
+    ngx_module_order="$ngx_module_name ngx_http_lua_module"
```

Comment:
This looks wrong to me since this module does not really depend on `ngx_lua` (though its test suite does).